app/persistence/transfer/transfer.go

```diff
@@ -14,22 +14,20 @@
  * limitations under the License.
  */
 
-package transaction
+package transfer
 
 import (
 	"errors"
-	"fmt"
-	"github.com/limechain/hedera-eth-bridge-validator/app/helper"
-	"github.com/limechain/hedera-eth-bridge-validator/app/helper/ethereum"
+	"github.com/limechain/hedera-eth-bridge-validator/app/persistence/entity"
 	"github.com/limechain/hedera-eth-bridge-validator/config"
 	"github.com/limechain/hedera-eth-bridge-validator/proto"
 	log "github.com/sirupsen/logrus"
 	"gorm.io/gorm"
 )
 
-// Enum Transaction Statuses
+// Enum Transfer Statuses
 const (
```

Comment:
Since we are referring to the statuses outside this file, wouldn't it be better if we place them in the `entity` package -> `transfer`